gensim/corpora/svmlightcorpus.py

```diff
@@ -112,8 +112,8 @@ def save_corpus(fname, corpus, id2word=None, labels=False, metadata=False):
         """
         logger.info("converting corpus to SVMlight format: %s", fname)
 
-        if type(labels) == np.ndarray:
-            labels = labels.tolist()    
+        if isinstance(labels, np.ndarray):
```

Comment:
Isn't better to just call `list(labels)` without any type cheks? In this case, `labels` can be any iterable (and no need to check types).